plugins/michi/skills/create-design/SKILL.md

```
@@ -140,6 +140,27 @@ project.json のフェーズを更新します：
+### 推奨: TODO抽出
+
+設計書からリスクや不明点を抽出してTODO管理を更新します。
+
+`manage-todos` スキルを使用してTODOを抽出します：
+
+```bash
+/michi manage-todos scan {pj-name}
+```
+
+または自動発動：
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)